tests/integration/metrics/column/test_column_values_match_regex_count.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+
+from great_expectations.datasource.fluent.interfaces import Batch
+from great_expectations.metrics.column.column_values_match_regex_count import (
+    ColumnValuesMatchRegexCount,
+    ColumnValuesMatchRegexCountResult,
+)
+from tests.integration.conftest import parameterize_batch_for_data_sources
+from tests.metrics.conftest import (
+    ALL_DATA_SOURCES,
+    PandasDataFrameDatasourceTestConfig,
+    PandasFilesystemCsvDatasourceTestConfig,
+    SnowflakeDatasourceTestConfig,
+)
+
+COLUMN_NAME = "whatevs"
+
+DATA_FRAME = pd.DataFrame({COLUMN_NAME: ["abc", "def", "ghi", "1ab2", "1ab3", None]})
+
+ALL_DATA_SOURCES_EXCEPT_SNOWFLAKE = [
+    datasource
+    for datasource in ALL_DATA_SOURCES
+    if not isinstance(datasource, SnowflakeDatasourceTestConfig)
+]
+
+ALL_DATA_SOURCES_EXCEPT_PANDAS = [
```

Comment:
SQL_DATA_SOURCES + SPARK_DATA_SOURCES is probably the easier way to do this, but potayto potahto